### ISSUE TITLE
Missing import Types

### ISSUE THREAD

**Issue by andreyvital** [dotansimha/graphql-code-generator#2276]
**Describe the bug**
 
Doesn't import "Types" if you spread only fragments

**To Reproduce**

1. My GraphQL operations:

```TSX
export default gql`
  query AssetTickerBySymbol($tickerSymbol: String!) {
    assetByTickerSymbol(tickerSymbol: $tickerSymbol) {
      ... on A {
        ...FragmentA
      }
      ... on B {
        ...FragmentB
      }
      ... on C {
        ...FragmentC
      }
    }
  }

  ${FragmentA}
  ${FragmentB}
  ${FragmentC}
`;
```

2. My `codegen.yml` config file:

```yml
schema: http://localhost:25256

generates:
  types/introspection-result.ts:
    plugins:
      - "fragment-matcher"

  types/graphql-types.ts:
    plugins:
      - "typescript"
      - "typescript-operations"

    config:
      typesPrefix: I
      immutableTypes: false
      skipTypename: false
      avoidOptionals: true
      nonOptionalTypename: true

  .:
    documents: "lib/{queries,fragments}/**/*.ts"
    preset: near-operation-file

    presetConfig:
      extension: .generated.ts
      baseTypesPath: ./types/graphql-types.ts

    plugins:
      - typescript-operations

    config:
      typesPrefix: I
      avoidOptionals: true
      nonOptionalTypename: true
```

**Expected behavior**
As you can see, I don't have any selection other than the fragments' spread. The resulting file (near-operation-file) doesn't generate the `import * as Types` accordingly. It needs to be generated because the argument `tickerSymbol` refers to a String:

```TSX
export type IAssetTickerBySymbolQueryVariables = {
  tickerSymbol: Types.Scalars["String"];
};
```

I guess it is because of this function: https://github.com/dotansimha/graphql-code-generator/blob/master/packages/presets/near-operation-file/src/utils.ts#L69-L85

It isn't accounting for arguments and is probably failing to detect fragment spreads. Hence we don't get the import: https://github.com/dotansimha/graphql-code-generator/blob/master/packages/presets/near-operation-file/src/index.ts#L170-L172. I could be wrong, though. Just a guesss!

Thanks for all the work. Love this project.

**Comment by dotansimha** [comment]
Hi @andreyvital . Which version of the codegen and plugins do you use? I think we already solved this before.

**Comment by andreyvital** [comment]
```
"@graphql-codegen/cli": "^1.4.0"
"@graphql-codegen/fragment-matcher": "^1.4.0"
"@graphql-codegen/near-operation-file-preset": "^1.4.0"
"@graphql-codegen/typescript": "^1.4.0"
"@graphql-codegen/typescript-operations": "^1.4.0"
```

**Comment by dotansimha** [comment]
@andreyvital I fixed this issue in: https://github.com/dotansimha/graphql-code-generator/commit/9cb9000a5e84a708e6814319c6bc3a7bef17743a
Can you please try the latest alpha? `1.5.1-alpha-84a08ebe.27`

**Comment by andreyvital** [comment]
hey sorry @dotansimha, missed that one. I'll give it a whirl.

**Comment by dotansimha** [comment]
Fixed in 1.6.0
